app/helpers/content_helper.rb

```diff
@@ -1,7 +1,7 @@
 module ContentHelper
   def render_content_page(page_name)
     raw_content = File.read("app/views/content/#{page_name}.md")
-    content_with_erb_tags_replaced = ApplicationController.renderer.render(inline: raw_content)
+    content_with_erb_tags_replaced = ApplicationController.renderer.render(inline: raw_content, with_toc_data: true)
```

Comment:
This is `ApplicationController::Renderer` rather than `RedCarpet` — I don't think `with_toc_data` will do anything here? https://api.rubyonrails.org/classes/ActionController/Renderer.html#method-i-render

Reply:
Well, it worked! Or maybe it was working already… I can double check.

Reply:
Ha! Turns out this was enabled anyway… just didn’t need to add manual ids in the markdown document.